modules/react/common/lib/utils/components.ts

```diff
@@ -600,7 +600,7 @@ export const createComponent =
  */
 export const createElemPropsHook =
   <TModelHook extends (config: any) => Model<any, any>>(modelHook: TModelHook) =>
-  <PO extends {}, PI extends {}>(
+  <const PO extends {}, const PI extends {}>(
```

Comment:
it's not happy about these const in here for some reason

Reply:
The dependency checker doesn't understand the new syntax... It uses an AST parser to function.